lib/fastlane/plugin/fueled/actions/define_versions_android.rb

```diff
@@ -13,7 +13,8 @@ def self.run(params)
         Actions.lane_context[SharedValues::FUELED_BUILD_NUMBER] = Helper::FueledHelper.new_build_number
         # Short Version
         current_short_version = Helper::FueledHelper.short_version_from_tag
-        if current_short_version.split('.').first.to_i >= 1
+        disable_version_limit = ENV['DISABLE_VERSION_LIMIT'] == 'true'
```

Comment:
Shouldn't this be part of the `available_options` ? This will serve as documentation I suppose, and could even provide an explicit default value.

Reply:
I'll update it soon with the same